src/index.js

```diff
@@ -1,6 +1,4 @@
-exports.install = function (Vue, options = {}) {
-  const defaultOptions = { messageNoData: '', classNoData: 'ct-nodata' }
-  options = Object.assign({}, defaultOptions, options)
+exports.install = function (Vue) {
```

Comment:
Yes, but I wouldn't use it as a default for `noDataOptions`.
I'd use a computed property which checks if `options` is set and uses that instead of the props in case it is set.
We'd need to make some changes to the code.

Reply:
I'm ok to test your suggestions

Reply:
> I'm ok to test your all suggestions

Great. Would you like me to update the PR with the latest changes we discussed?

Reply:
Yes please, it's easier to test for me.